src/WordPress.php

```diff
@@ -1296,6 +1296,49 @@ public function reload( $args, $assoc_args, $whitelisted_containers = [], $reloa
 	}
 
 
+	/**
+	 * Publishes site online using ngrok.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <site-name>
+	 * : Name of website.
+	 *
+	 * [--disable]
+	 * : Take online link down.
+	 *
+	 * [--refresh]
+	 * : Refresh site publish if link has expired.
+	 *
+	 * [--token=<token>]
+	 * : ngrok token.
+	 */
+	public function publish( $args, $assoc_args ) {
+
+		parent::publish( $args, $assoc_args );
+
+		$disable = \EE\Utils\get_flag_value( $assoc_args, 'disable', false );
```

Comment:
Please use nemespace and then it's a function. Don't use it directly. we are already refactoring this and that won't matter if we keep adding new code with same issue.

Reply:
Updated :+1: